### PR TITLE
Master double menu yti

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tivisse** [comment]
@robodoo r+

File: addons/hr_contract/report/hr_contract_history_report_views.xml [hunk 1]
```diff
@@ -197,13 +197,7 @@
         id="hr_menu_contract_history"
         action="hr_contract_history_view_list_action"
         parent="hr.menu_hr_employee_payroll"
+        name="Contracts"
```
Comment (tde-banana-odoo) [inline addons/hr_contract/report/hr_contract_history_report_views.xml:200]: ''.join("C o n t r a c t s".split(' '))

**Review by tde-banana-odoo — CHANGES_REQUESTED** [review]

File: addons/hr_contract/report/hr_contract_history_report_views.xml (hunk 1 above)
Comment (mart-e) [reply addons/hr_contract/report/hr_contract_history_report_views.xml:200]:
```py
''.join(_("C o n t r a c t s").split(' '))
```
translations please !

Comment (tivisse) [reply addons/hr_contract/report/hr_contract_history_report_views.xml:200]: Ok done.

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**